test/test_session.cpp

```diff
@@ -333,7 +333,7 @@ TORRENT_TEST(save_state_fingerprint)
 auto const count_dht_inits = [](session& ses)
 {
 	int count = 0;
-	int num = 120; // this number is adjusted per version, an estimate
+	int num = 200; // this number is adjusted per version, an estimate
```

Comment:
this seems unrelated

Reply:
I was having problems in my env that required this change, will revert it